WooCommerce/WooCommerceTests/ViewModels/OrderListCellViewModelTests.swift

```diff
@@ -0,0 +1,56 @@
+import XCTest
+import TestKit
+
+@testable import Yosemite
+@testable import WooCommerce
+
+final class OrderListCellViewModelTests: XCTestCase {
+
+    func test_dateCreated_does_not_include_the_year_for_dates_with_the_same_year() {
+        // Given
+        let order = MockOrders().sampleOrderCreatedInCurrentYear()
+        let viewModel = OrderListCellViewModel(order: order, status: nil)
+        let expectedYearString = DateFormatter.yearFormatter.string(from: order.dateCreated)
+
+        // Then
+        let formatted = viewModel.dateCreated
+
+        XCTAssertFalse(formatted.contains(expectedYearString))
+    }
+
+    func test_dateCreated_includes_the_year_for_dates_that_are_not_in_the_same_year() {
+        // Given
+        let order = MockOrders().sampleOrder()
+        let viewModel = OrderListCellViewModel(order: order, status: nil)
+        let expectedYearString = DateFormatter.yearFormatter.string(from: order.dateCreated)
+
+        // Then
+        let formatted = viewModel.dateCreated
+
+        XCTAssertTrue(formatted.contains(expectedYearString))
+    }
+
+    func test_OrderStatus_is_used_when_passed_to_view_model() {
+        // Given
+        let order = MockOrders().sampleOrder()
+        let orderStatus = OrderStatus(name: UUID().uuidString,
+                                      siteID: 0,
+                                      slug: UUID().uuidString,
+                                      total: 0)
+        let viewModel = OrderListCellViewModel(order: order, status: orderStatus)
```

Comment:
Thanks for using Given-When-Then! 

I would say this is the When part. 🙂 

```suggestion

        // When
        let viewModel = OrderListCellViewModel(order: order, status: orderStatus)
```

Reply:
Thank you! Updated :-)